docs/content/deprecated/Dialog.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: Dialog
+title: Dialog (legacy)
```

Comment:
Do you think using `legacy` will add to any confusion about which component we are talking about?

Reply:
If anything, I think it makes it clearer